tools/tck-integrity-tests/src/test/scala/org/opencypher/tools/tck/ScenarioFormatChecker.scala

```diff
@@ -27,19 +27,45 @@
  */
 package org.opencypher.tools.tck
 
-import io.cucumber.datatable.DataTable
-import org.junit.platform.runner.JUnitPlatform
-import org.junit.runner.RunWith
-import org.scalatest.funsuite.AnyFunSuite
+import org.opencypher.tools.tck.api.Scenario
+import org.opencypher.tools.tck.api.groups.ContainerGroup
+import org.opencypher.tools.tck.api.groups.Group
+import org.opencypher.tools.tck.api.groups.Item
+import org.opencypher.tools.tck.api.groups.Tag
+import org.opencypher.tools.tck.api.groups.TckTree
+import org.opencypher.tools.tck.api.groups.Total
+import org.scalatest.OptionValues
+import org.scalatest.ParallelTestExecution
+import org.scalatest.funspec.AsyncFunSpec
 import org.scalatest.matchers.should.Matchers
 
-import scala.collection.convert.DecorateAsJava
+import scala.concurrent.Future
 
-@RunWith(classOf[JUnitPlatform])
-abstract class TckTestSupport extends AnyFunSuite with Matchers with DecorateAsJava {
+trait ScenarioFormatChecker extends AsyncFunSpec with Matchers with OptionValues with ValidateScenario with ParallelTestExecution {
 
-  def tableOf(strings: Seq[String]*): DataTable = {
-    DataTable.create(strings.map(_.toList.asJava).toList.asJava)
-  }
+  def create(scenarios: Seq[Scenario]): Unit = {
+
+    implicit val tck: TckTree = TckTree(scenarios)
 
+    def spawnTests(currentGroup: Group): Unit = {
+      currentGroup match {
+        case Total =>
+          Total.children.foreach(spawnTests)
+        case _: Tag => Unit
+        case g: ContainerGroup =>
+          describe(g.description) {
+            g.children.foreach(spawnTests)
```

Comment:
Add
```suggestion
            g.children.toSeq.sorted.foreach(spawnTests)
```
to get sorted output.